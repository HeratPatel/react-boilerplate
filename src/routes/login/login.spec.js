import React from 'react';
import { shallow } from 'enzyme';
import Login from './login';

describe('<Login />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper).toMatchSnapshot();
  });
});
