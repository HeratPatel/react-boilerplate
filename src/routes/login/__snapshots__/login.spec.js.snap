// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> Rendering 1`] = `
<section>
  <h1>
    Login Page
  </h1>
</section>
`;
